### PR TITLE
Detect prev. asym. deposit to disable sym. deposit

### LINKED ISSUE #1828: [Deposit] Pending RUNE has been ignored
## Issue
ADD liquidity (sym.) for `XRUNE/RUNE` is failing for following paired addresses (dev wallet of ASGDX)
- RUNE tthor13gym97tmw3axj3hpewdggy2cr288d3qffr8skg 
- ETH.XRUNE 0x33292c1d02c432d323fb62c57fb327da45e1bdde
- Failing tx https://viewblock.io/thorchain/tx/98A9FD49E0BBCA02483E6258E18BA81DF58A26BD49A2B4E68EC2309FD4D63259?network=testnet" - Error `mismatch of asset address`

That's because `tthor13gym97tmw3axj3hpewdggy2cr288d3qffr8skg` had been used to add liquidity to ETH.XRUNE-0X8626DB1A4F9F3E1002EEB9A4F3C6D391436FFC23 pool asymmetrically before, see https://testnet.thornode.thorchain.info/thorchain/pool/ETH.XRUNE-0X8626DB1A4F9F3E1002EEB9A4F3C6D391436FFC23/liquidity_providers (search for `tthor13gym97tmw3axj3hpewdggy2cr288d3qffr8skg`).

Screenshot from 2021-10-06 09-47-57

Pending 

## Expected behavior

- Show "Pending assets" warning in case of pending RUNE (not just for pending asset)
- Disable ADD liquidity.

### PR DESCRIPTION
**Pull request by veado**
- [x] `useLiquidityProviders`
- [x] Check status of `asym` deposits in `SymDeposit`
- [x] Update `i18n`

https://user-images.githubusercontent.com/61792675/137783699-3c189b97-e1ea-4bc5-a517-ef059d3466d2.mp4

Close #1828

### CODE REVIEW

File: src/renderer/components/deposit/add/Deposit.styles.ts [hunk 1]
```diff
@@ -146,7 +146,7 @@ export const SubmitButton = styled(UIButton).attrs({
   margin-bottom: 20px;
```
Comment (veado) [inline src/renderer/components/deposit/add/Deposit.styles.ts:146]: Just renaming styles in `Deposit.styles.ts` - changes can be ignored

File: src/renderer/components/deposit/add/PendingAssetsWarning.stories.tsx [hunk 2]
```diff
@@ -6,7 +6,7 @@ import * as FP from 'fp-ts/lib/function'
```
Comment (veado) [inline src/renderer/components/deposit/add/PendingAssetsWarning.stories.tsx:6]: Just moving existing content of `Deposit.subcomponents.stories` into `PendingAssetsWarning.stories.tsx` - changes can be ignored

File: src/renderer/components/deposit/add/PendingAssetsWarning.tsx [hunk 3]
```diff
@@ -22,16 +22,16 @@ const AssetIconAmount: React.FC<AssetIconAmountProps> = (props): JSX.Element =>
     loading
```
Comment (veado) [inline src/renderer/components/deposit/add/PendingAssetsWarning.tsx:22]: Just moving existing content of `Deposit.subcomponents` into `PendingAssetsWarning` and using renamed style names - changes can be ignored

File: src/renderer/services/thorchain/thorNode.ts [hunk 4]
```diff
@@ -142,7 +139,11 @@ const getLiquidityProviders = ({ asset, network }: GetLiquidityProvidersParams):
         RxOp.map(A.filterMap(FP.flow(LiquidityProviderIO.decode, O.fromEither))),
         RxOp.map(RD.success),
         liveData.map(
+          // transform LiquidityProviderIO -> LiquidityProvider
           A.map((provider): LiquidityProvider => {
+            console.log('provider -> ', poolString)
```
Comment (gromxyz) [inline src/renderer/services/thorchain/thorNode.ts]: Leftover console.logs

**Review by gromxyz — DISMISSED** [review]

File: src/renderer/services/thorchain/thorNode.ts (hunk 4 above)
Comment (veado) [reply src/renderer/services/thorchain/thorNode.ts]: Good find, will remove it.

**Pull request merged**